app/models/request_type/validator.rb

```diff
@@ -70,10 +70,14 @@ def validate?(_value)
     def default
       nil
     end
+
+    def valid_options
+      []
+    end
   end
 
-  belongs_to :request_type
-  validates :request_type, :request_option, :valid_options, presence: true
```

Comment:
This is related to some of the cop issues @stevieing ran into. Migrating the validators to the new approach while I'min this class. Essentially no behaviour changes, but future proofs us for future Rails upgrades.